.github/workflows/android-e2e.yml

```diff
@@ -3,6 +3,7 @@ name: Android e2e tests
 on: [pull_request]
```

Comment:
I think we need to add this to the android task so that it runs again when the PR is marked as ready

```
on:
  pull_request:
    types: [opened, synchronize, reopened, ready_for_review]
```

I can see the iOS e2e ran, but not the android one.

Reply:
@janicduplessis I just pushed the changes ! :)